src/components/filter/criteriaBased/CriteriaBasedFilterEditionDialog.tsx

```diff
@@ -46,15 +46,15 @@ const formSchema = yup
     .required();
 
 export interface CriteriaBasedFilterEditionDialogProps {
-    id: string;
+    id: string | undefined | null;
```

Comment:
I don't think that `saveFilter()` support the now _null_ or _undefined_ `id` of `frontToBackTweak()`...

Reply:
In practice, id will not be null or undefined when saveFilter will be called ...